src/modules/ekf2/EKF/yaw_update.cpp

```diff
@@ -0,0 +1,147 @@
+/****************************************************************************
```

Comment:
Should we rather call this file `yaw_fusion.cpp` to follow the pattern?

Reply:
renamed